CHANGELOG.md

```diff
@@ -81,12 +82,15 @@
 - [0.2.0](#020)
 - [0.1.0](#010)
 
-## Unreleased
+## [v0.66.0]
+
+> Release date: 2025/05/12
 
 - Implement `FillID` method for plugins. It generates ID by plugin's name,
   service, route, consumer, and consumer group (if exists).
   [#542](https://github.com/Kong/go-kong/pull/542)
 
+
```

Comment:
```suggestion
```

I don't think we need this double new line here, right?

Reply:
This was used to trigger a CI re-run. Resolved now.